extensions/amp-graphiq/amp-graphiq.md

```diff
@@ -0,0 +1,76 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-graphiq"></a> `amp-graphiq`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Displays a [Graphiq embed](https://www.graphiq.com/).</td>
```

Comment:
This needs to be in HTML instead of markdown because it's inside of HTML.  So please change to:

`<a href="https://www.graphiq.com">Graphiq embed</a>`